packages/integration-tests/src/EntityManager.createOrUpdatePartial.test.ts

```diff
@@ -215,6 +215,57 @@ describe("EntityManager", () => {
     expect(await countOfBookToTags()).toEqual(0);
   });
 
+  it("collections can incrementally remove children", async () => {
+    // Given an book with two tags
+    await insertAuthor({ first_name: "a1" });
+    await insertBook({ title: "b1", author_id: 1 });
+    await insertTag({ name: "t1" });
+    await insertTag({ name: "t2" });
+    await insertBookToTag({ tag_id: 1, book_id: 1 });
+    await insertBookToTag({ tag_id: 2, book_id: 1 });
+    const em = newEntityManager();
+    // When we incrementally remove a single tag
+    await em.createOrUpdatePartial(Book, { id: "b:1", tags: [{ id: "t:2", op: "remove" }] });
+    await em.flush();
+    // Then we removed only that one m2m row
```

Comment:
Can we check that this has deleted the proper line? Since we're going from 2 to 1... it's not CERTAIN that this deleted the right one instead of deleting "all but"

Reply:
Yep, done